2418.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {string[]} names
+ * @param {number[]} heights
+ * @return {string[]}
+ */
+var sortPeople = function (names, heights) {
+  let sorted = [...heights].sort((a, b) => b - a);
+  let result = [];
```

Comment:
```suggestion
  const sorted = [...heights].sort((a, b) => b - a);
  const result = [];
```
Good to use `const` here 👍

Reply:
Well done for avoiding side effects 👍